cosmotheka/mappers/mapper_ROSAT.py

```diff
@@ -113,17 +157,18 @@ def _get_mask(self):
             msk = rotate_mask(msk, self.rot, binarize=True)
             msk = hp.ud_grade(msk, nside_out=self.nside)
             mask *= msk
+        if self.mask_ps:
+            # Mask point sources
+            ps_mask = hp.ud_grade(self._get_ps_mask(),
```

Comment:
Duplicated of L.128? Maybe `ps_mask` can be stored to avoid changing the resolution twice?

Reply:
No, we want to remove the point sources in the map at the native resolution (not at the potentially downgraded one). Remember it's the same thing we do for point sources in the SPT maps. It's also the reason why we want `_get_ps_mask` to return the mask in the native high resolution.

Reply:
I understand that, but L128 is exactly the same this line, as far I can see

Reply:
fine